moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -122,6 +123,7 @@ void MotionPlanningFrame::sceneScaleChanged(int value)
           ps->getWorldNonConst()->addToObject(scaled_object_->id_, shapes::ShapeConstPtr(s),
                                               scaled_object_->shape_poses_[i]);
         }
+        ps->getWorldNonConst()->setSubframesOfObject(scaled_object_->id_, subframes);
```

Comment:
This might be a matter of personal preference, but shouldn't the subframes be scaled as well?
I guess there is no reasonable way to do that until your main pose request is merged...
Could you add a TODO in the code here though?

Reply:
Good idea